src/MetaModels/Attribute/Timestamp/Timestamp.php

```diff
@@ -58,7 +58,8 @@ public function getFieldDefinition($arrOverrides = array())
         $strDateType                       = $this->getDateTimeType();
         $arrFieldDef                       = parent::getFieldDefinition($arrOverrides);
         $arrFieldDef['eval']['rgxp']       = $strDateType;
-        $arrFieldDef['eval']['datepicker'] = !empty($arrFieldDef['eval']['readonly'])?false:true;
+        $arrFieldDef['eval']['datepicker'] = empty($arrFieldDef['eval']['readonly']);
+        $arrFieldDef['eval']['tl_class']  .= ' wizard';
```

Comment:
I fail to see the benefit of adding an icon which will never be clickable at all.